src/rules/no_duplicate_case.rs

```diff
@@ -30,6 +30,44 @@ impl LintRule for NoDuplicateCase {
     let mut visitor = NoDuplicateCaseVisitor::new(context);
     visitor.visit_module(module, module);
   }
+
+  fn docs(&self) -> &'static str {
+    r#"Disallows using the same case clause in a switch statement more than once
+
+When you reuse a case test expression in a `switch` statement, the duplicate case is
+never be reached meaning this is almost always a bug.
```

Comment:
```suggestion
When you reuse a case test expression in a `switch` statement, the duplicate case will
never be reached meaning this is almost always a bug.
```

I guess you wanted to write like this?